dissect/target/plugin.py

```diff
@@ -1074,7 +1075,8 @@ def add_to_result(func: PluginFunction) -> None:
                 matches = True
                 add_to_result(
                     PluginFunction(
-                        name=index_name,
+                        name=f"{func['namespace']}.{method_name}" if func["namespace"] else method_name,
```

Comment:
This might cause unwanted behaviour in query.py, because the names are now no longer unique unlike the paths?
```

if args.targets:
            for target in args.targets:
               ....
                for func in funcs:
                    collected_plugins[func.name] = func.plugin_desc
        else:
           ....
            for func in funcs:
                collected_plugins[func.name] = func.plugin_desc
```